pmd-visualforce/src/test/java/net/sourceforge/pmd/lang/vf/rule/security/VfHtmlXssStyleTagUrlPatternMatchingTest.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.vf.rule.security;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+/***
+ * Unit tests to focus on regex pattern used to identify URL methods within style tags
+ */
+public class VfHtmlXssStyleTagUrlPatternMatchingTest {
```

Comment:
Unit tests to verify distinction between url-based and non-url-based values in style tags.